src/test/java/io/openliberty/tools/intellij/it/UIBotTestUtils.java

```diff
@@ -120,8 +120,9 @@ public static void importProject(RemoteRobot remoteRobot, String projectsPath, S
             // From the project frame.
             ProjectFrameFixture projectFrame = remoteRobot.find(ProjectFrameFixture.class, Duration.ofSeconds(30));
             commonFixture = projectFrame;
+            clickOnMainMenu(remoteRobot);
             ComponentFixture fileMenuEntry = projectFrame.getActionMenu("File", "10");
-            fileMenuEntry.click();
+            fileMenuEntry.moveMouse();
```

Comment:
Can you explain this change?

Reply:
https://github.com/user-attachments/assets/5730077c-5c92-4b2c-a5ef-c764a69202ed

There is an issue with Linux where, as shown in the video, when we try to access 'Copy' via the Edit menu, clicking on it causes the popup window to disappear.